### DISCUSSION TITLE
Transactions

### DISCUSSION THREAD

**Discussion by cemoktra** [SeaQL/sea-orm#198] [Q&A] [answered]
Currently i can have a transaction of multiple statements, but i cannot do some other work in between. For example
- start transaction
- insert something
- no DB operation, but required for following update
- update another table
- commit transaction

**Comment by zackangelo** [comment]
Transactions don't seem to be addressed at all in the docs. If they're intentionally omitted from SeaORM's design it would be helpful to mention it. If more support is planned in the future, it might be worth it to mention that instead. 

If there's a current vision around this I'd be happy to take a stab at it or open a docs PR! 😊

**Reply by billy1624** [reply]
Hi, transaction is still under development. We will update the docs when it's released :)

**Reply by billy1624** [reply]
A docs PR will be welcome! But first we have to make the transaction API stable.

**Comment by zackangelo** [comment]
Oh, just noticed this PR, which seems relevant: https://github.com/SeaQL/sea-orm/pull/222

**Reply by tyt2y3** [reply]
Yes I have been focusing on it. API is pretty settled, just that more testing is needed.

Do you have any suggestions on test cases?

**Comment by tyt2y3** [comment, marked as answer]
https://www.sea-ql.org/SeaORM/blog/2021-10-15-whats-new-in-0.3.0
